be/src/vec/sink/vtablet_finder.cpp

```diff
@@ -95,8 +95,22 @@ Status OlapTabletFinder::find_tablets(RuntimeState* state, Block* block, int row
     if (_find_tablet_mode == FindTabletMode::FIND_TABLET_EVERY_ROW) {
         _vpartition->find_tablets(block, qualified_rows, partitions, tablet_index);
     } else {
+        // for random distribution
```

Comment:
I think the following code can be removed?
https://github.com/apache/doris/blob/4230b8c36cac9636199eebb7c0adf82ee176b46e/be/src/exec/tablet_info.h#L211-L221

Reply:
cannot, we will find tablet index here every batch